testing/src/test/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -364,6 +366,150 @@ public void deleteTable() throws IOException {
         HttpStatus.SC_NO_CONTENT);
   }
 
+  @Test
+  public void createIndex() throws IOException {
+    createKeyspace(keyspaceName);
+    String tableName = "tbl_createtable_" + System.currentTimeMillis();
+    createTestTable(
+        tableName,
+        Arrays.asList("id text", "firstName text", "email list<text>"),
+        Collections.singletonList("id"),
+        null);
+
+    IndexAdd indexAdd = new IndexAdd();
+    indexAdd.setTable(tableName);
+    indexAdd.setColumn("firstName");
+    indexAdd.setName("test_idx");
+    indexAdd.setIfNotExists(false);
+
+    String body =
+        RestUtils.post(
+            authToken,
+            String.format("%s:8082/v2/schemas/keyspaces/%s/indexes", host, keyspaceName),
+            objectMapper.writeValueAsString(indexAdd),
+            HttpStatus.SC_CREATED);
+    SuccessResponse successResponse =
+        objectMapper.readValue(body, new TypeReference<SuccessResponse>() {});
+    assertThat(successResponse.getSuccess()).isTrue();
```

Comment:
Would it make sense to also validate that the index exists via a CQL query?

Reply:
Yes, gonna do that.

Reply:
@dimas-b Please, do you know how could I plug a CQL call inside those tests?

Reply:
You can either do it this way https://github.com/stargate/stargate/blob/master/testing/src/test/java/io/stargate/it/http/GraphqlTest.java#L183 or you can use annotations like https://github.com/stargate/stargate/blob/master/testing/src/test/java/io/stargate/it/cql/WarningsTest.java#L22